HACKING.md

```diff
@@ -15,7 +15,11 @@ The resulting images can then be pushed to a Docker registry.
 
     $ hack/build-images.sh
 
-Note: To build the base and release images, run:
+NOTE:  If your code changes impact the functions and/or methods that run in the OpenShift builder pod and drive an OpenShift image build, and assuming you
```

Comment:
i'd split this into two comments.

1) under hack/build-release, note that only committed code is built.
2) under hack/build-images note that you need to do this if your code is part of any of the images openshift runs internally such as origin-sti-builder, origin-docker-builder, origin-deployer, etc.